### PR TITLE
Styled components

### PR DESCRIPTION
**Pull request by alexzhaohong**
Visual change only.

Screen Shot 2020-05-11 at 10 30 44 AM

Screen Shot 2020-05-11 at 10 31 52 AM

### CODE REVIEW

File: client/src/components/RatingTallyEntry.jsx [hunk 1]
```diff
@@ -1,4 +1,75 @@
...
+margin: 0 10px;
+background-color: #767677;
+border-radius: 2px;
+font-size: 16px;
+height: 4px;
+-webkit-font-smoothing: antialiased;
+`;
+const Fill = styled(Bar)`
+margin: 0;
+width: ${props => `${props.width}%`};
```
Comment (karoonp) [inline client/src/components/RatingTallyEntry.jsx:61]: this is nice!

File: client/src/components/RatingTallyEntry.jsx [hunk 2]
```diff
@@ -8,20 +79,18 @@ class ReviewListEntry extends React.Component {
...
+      </Distribution>
     );
   }
 }
 
-export default ReviewListEntry;
\ No newline at end of file
+export default ReviewListEntry;
+
+// ========-------
```
Comment (karoonp) [inline client/src/components/RatingTallyEntry.jsx:96]: can be removed if not needed.

File: client/src/components/RatingTally.jsx [hunk 3]
```diff
@@ -1,33 +1,65 @@
...
-        <div>ratings & reviews</div>
-        <div> 4.7 6860 Reviews</div>
-        <div>rating breakdown</div>
-        <RatingTallyEntry />
-        <RatingTallyEntry />
-        <RatingTallyEntry />
-        <RatingTallyEntry />
-        <RatingTallyEntry />
-      </Fragment>
+      <>
```
[the remark this answers is not in the document]
Comment (alexzhaohong) [reply client/src/components/RatingTally.jsx:46]: <></> is the same as <Fragment></Fragment>. It's React interpretation

File: client/src/components/RatingTallyEntry.jsx (hunk 1 above)
Comment (alexzhaohong) [reply client/src/components/RatingTallyEntry.jsx:61]: It was a headache...

File: client/src/components/RatingTallyEntry.jsx (hunk 2 above)
Comment (alexzhaohong) [reply client/src/components/RatingTallyEntry.jsx:96]:
```suggestion
```

**Pull request merged**